app/src/main/res/layout/fragment_maps.xml

```diff
@@ -47,21 +47,20 @@
             <!-- Media -->
             <ImageView
                 android:id="@+id/restaurant_img"
-                android:layout_width="wrap_content"
-                android:layout_height="145dp"
-                app:srcCompat="@drawable/earth"
-                android:elevation="2dp"
+                android:layout_width="600dp"
```

Comment:
I doubt that setting the fixed width would be helpful. Think of the responsiveness in multiple screen-size devices. Setting this `match_parent` would help in this case